schemas/container.json

```diff
@@ -51,6 +51,9 @@
                 },
                 "negate": {
                   "type": "boolean"
+                },
+                "sequence_timeout" : {
```

Comment:
Can we just call it `timeout`? Whether the timeout is used in a sequencing context doesn't seem relevant. Regardless of what we end up with, it needs to be camelCase.

Reply:
Thanks, I renamed it to `timeout`.